docs/user-guide/install-customization.md

```diff
@@ -105,6 +108,42 @@ curl \
 curl --header "Authorization: Bearer $TOKEN" "http://$ASSISTED_SERVICE_IP:$ASSISTED_SERVICE_PORT/api/assisted-install/v2/clusters/$CLUSTER_ID/install-config"
 ```
 
+### Exclude one or more optional components (capabilities)
+
+Since [OpenShift 4.12](https://github.com/openshift/enhancements/blob/master/enhancements/installer/component-selection.md#resource-management), it is possible to customize the install config to disable some components.
+
+The features (`baremetal-operator`, `marketplace`, `Console`, `Insights`, `Storage`, `CSISnapshot` and `openshift-samples-content`) can be disabled by setting the `baselineCapabilitySet` and `addittionalEnabledCapabilities` parameters in the `install-config.yaml`.
```

Comment:
@iranzo where do you stand with this doc? Also, can you please try to remove console and see whether we still can manage to isntall?

Reply:
Sure, I'll try to remove console and validate installation

Reply:
any update?

Reply:
I would say that @javipolo did test it as I was on leave and his feedback was added to the PR. 

Can you confirm Javi?

Reply:
Yep, I did try it and without the console component, AI got stuck on 'Finalizing' and ultimately failed because of timeout

@jhernand was working on fixing that issue

Reply:
are we waiting for the fix in order to get this piece merged?
I believe we can merge it already, as it won't get published in the next few days?

Reply:
That is addressed in #4594, reviews are welcome.